packages/salesforcedx-vscode-expanded/README.md

```diff
@@ -39,21 +39,21 @@ To report issues with Salesforce Extensions for VS Code, open a [bug on GitHub](
 
 The Salesforce Extension Pack extension installs these extensions.
 
-- [Salesforce CLI Integration](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-core)  
+- [Salesforce CLI Integration](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-core)
    This extension (`salesforcedx-vscode-core`) interacts with Salesforce CLI to provide core functionality.
-- [Apex](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-apex)  
+- [Apex](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-apex)
    This extension (`salesforcedx-vscode-apex`) uses the Apex Language Server to provide features such as syntax highlighting and code completion.
-- [Apex Interactive Debugger](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-apex-debugger)  
+- [Apex Interactive Debugger](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode-apex-debugger)
```

Comment:
For some reason, there was a lot of trailing whitespace.  I use an extension which automatically trims lines of code, so there are a lot of changes here.  To see what was really changed by me, here in GitHub change the dif view setting to ignore whitespace changes.